build.gradle

```diff
@@ -123,17 +123,18 @@ subprojects {
 
                 // Test dependencies.
                 junit: 'junit:junit:4.11',
-                mockito: 'org.mockito:mockito-core:1.10.19'
+                mockito: 'org.mockito:mockito-core:1.10.19',
+                tcnative: 'io.netty:netty-tcnative:1.1.33.Fork3'
```

Comment:
I added tcnative because it would fail without it as:
```
simonma@simonma-linux:~/github/grpc-java$ ./run-test-server.sh 
Gradle is no longer run automatically. Make sure to run './gradlew installDist' or
'./gradlew :grpc-interop-testing:installDist' after any changes.

Using fake CA for TLS certificate. Test clients should expect host
*.test.google.fr and our test CA. For the Java test client binary, use:
--server_host_override=foo.test.google.fr --use_test_ca=true

Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/tomcat/jni/CertificateVerifier
	at io.netty.handler.ssl.SslContext.newServerContextInternal(SslContext.java:403)
	at io.netty.handler.ssl.SslContextBuilder.build(SslContextBuilder.java:207)
	at io.grpc.testing.integration.TestServiceServer.start(TestServiceServer.java:138)
	at io.grpc.testing.integration.TestServiceServer.main(TestServiceServer.java:76)
Caused by: java.lang.ClassNotFoundException: org.apache.tomcat.jni.CertificateVerifier
	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	... 4 more

```